doc/src/guides/solving/solving-guidance.md

```diff
@@ -0,0 +1,221 @@
+# Solving Guidance
+
+These guidelines apply to many types of solving.
+
+## Numeric Solutions
+
+### Numeric Solutions by Choice
```

Comment:
This is a string title name. It's not clear what "by choice" is supposed to mean here.